build/common.sh

```diff
@@ -95,9 +95,9 @@ kube::build::get_docker_wrapped_binaries() {
   ### in build/BUILD. And kube::golang::server_image_targets
   local targets=(
     cloud-controller-manager,"k8s.gcr.io/debian-base-${arch}:${debian_base_version}"
-    kube-apiserver,"k8s.gcr.io/debian-base-${arch}:${debian_base_version}"
-    kube-controller-manager,"k8s.gcr.io/debian-base-${arch}:${debian_base_version}"
-    kube-scheduler,"k8s.gcr.io/debian-base-${arch}:${debian_base_version}"
+    kube-apiserver,"gcr.io/distroless/static:latest"
+    kube-controller-manager,"gcr.io/distroless/static:latest"
+    kube-scheduler,"gcr.io/distroless/static:latest"
```

Comment:
Will this get the right architecture?

Reply:
I think all the arch will use the same distroless base image. Does distroless image need multi-arch support?

Reply:
I have no idea what it does here to be honest, but previously we explicitly selected an image matching the architecture we are building for.